uber/automated_emails.py

```diff
@@ -23,53 +25,211 @@ def __init__(self, model, subject, template, filter, *, sender=None, extra_data=
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
         self.ident = (ident or self.subject).format(EVENT_NAME=c.EVENT_NAME)
+        self.when = when
+
+        # after each daemon run, this will be set to the number of emails that would have been sent out but weren't
+        # because they were not marked as approved.  Useful as a metric of how many emails need human intervention in
+        # order to be approved for sending.
+        #
+        # A value of -1 means we haven't run yet, or email sending is disabled. Always check for > -1.
+        self.unapproved_emails_not_sent = -1
```

Comment:
ok cool agreed, I went ahead and changed to check for None.

one thing to be careful of is that None and 0 (which is a valid thing) are both false, so we have to explicitly check for ```if x != None``` instead of just saying like ```if x```. if we did just check for falsiness, we'd come back false for values that are 0, when we instead want to consider that a real value.